### DISCUSSION TITLE
Read text from Google Docs

### DISCUSSION THREAD

**Discussion by jordatormo** [pgmichael/wavenet-for-chrome#67] [General]
Would it be possible to read-to-speech a text inside a Google Doc. It looks like the right-click menu does not show the Start speaking button. 
The shortcut does not work either in Google Docs.
Any suggestions?
Thanks!

**Comment by pgmichael** [comment]
Not at the moment @jordatormo. Google docs has it's own rendering engine so the common browser APIs to retrieve the selection doesn't work. There might be some workarounds. Might look into it eventually.

You can subscribe to https://github.com/pgmichael/wavenet-for-chrome/issues/16
